types/LambdaBuffers/ApplicationTypes/Configuration.purs

```diff
@@ -0,0 +1,534 @@
+                        ((Data.Newtype.unwrap x1).voteFungibleCurrencySymbol)
+                    )
+                )
+                ( Prelude.(==) ((Data.Newtype.unwrap x0).voteFungibleTokenName)
+                    ((Data.Newtype.unwrap x1).voteFungibleTokenName)
+                )
+            )
+            ( Prelude.(==) ((Data.Newtype.unwrap x0).fungibleVotePercent)
+                ((Data.Newtype.unwrap x1).fungibleVotePercent)
+            )
+        )
+    )
+
+instance Ctl.Internal.ToData.ToData DynamicConfigDatum where
+  toData =
+    ( \x0 -> Ctl.Internal.Types.PlutusData.List
+        ( [ Ctl.Internal.ToData.toData ((Data.Newtype.unwrap x0).tallyValidator)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).treasuryValidator)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).configurationValidator)
+          , Ctl.Internal.ToData.toData ((Data.Newtype.unwrap x0).voteValidator)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).upgradeMajorityPercent)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).upgradeRelativeMajorityPercent)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).generalMajorityPercent)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).generalRelativeMajorityPercent)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).tripMajorityPercent)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).tripRelativeMajorityPercent)
+          , Ctl.Internal.ToData.toData ((Data.Newtype.unwrap x0).totalVotes)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).maxGeneralDisbursement)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).maxTripDisbursement)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).agentDisbursementPercent)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).proposalTallyEndOffset)
+          , Ctl.Internal.ToData.toData ((Data.Newtype.unwrap x0).tallyNft)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).voteCurrencySymbol)
+          , Ctl.Internal.ToData.toData ((Data.Newtype.unwrap x0).voteTokenName)
+          , Ctl.Internal.ToData.toData ((Data.Newtype.unwrap x0).voteNft)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).voteFungibleCurrencySymbol)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).voteFungibleTokenName)
+          , Ctl.Internal.ToData.toData
+              ((Data.Newtype.unwrap x0).fungibleVotePercent)
+          ]
+        )
+    )
+
+instance Ctl.Internal.FromData.FromData DynamicConfigDatum where
+  fromData =
+    ( \x0 -> LambdaBuffers.Runtime.Plutus.casePlutusData
```

Comment:
Was this generated?

Reply:
Exactly, these instances are generated with LambdaBuffers